src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -212,40 +246,23 @@ export default class GitOpsConfiguration extends Component<GitOpsProps, GitOpsSt
-                        <span className="tertiary-tab sso-icons">
-                            <aside className="login__icon-alignment"><GitHub /></aside>
-                            <aside className="login__text-alignment"> GitHub</aside>
-                            <div>
-                                {(this.state.lastActiveGitOp?.provider === GitProvider.Github) ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
-                            </div>
-                        </span>
-                    </label>
-                    <label className="tertiary-tab__radio">
-                        <input type="radio" name="status" value={GitProvider.GitLab} checked={this.state.tab === GitProvider.GitLab} onChange={this.handleGitopsTab} />
-                        <span className="tertiary-tab sso-icons">
-                            <aside className="login__icon-alignment"><GitLab /></aside>
-                            <aside className="login__text-alignment"> GitLab</aside>
-                            <div>
-                                {this.state.lastActiveGitOp?.provider === GitProvider.GitLab ? <aside className="login__check-icon"><img src={Check} /></aside> : ""}
-                            </div>
-                        </span>
-                    </label>
+                    <GitProviderTab tab={this.state.tab} handleGitopsTab={this.handleGitopsTab} lastActiveGitOp={this.state.lastActiveGitOp} provider={GitProvider.Github} gitops="GitHub" />
+                    <GitProviderTab tab={this.state.tab} handleGitopsTab={this.handleGitopsTab} lastActiveGitOp={this.state.lastActiveGitOp} provider={GitProvider.GitLab} gitops="GitLab" />
+                    <GitProviderTab tab={this.state.tab} handleGitopsTab={this.handleGitopsTab} lastActiveGitOp={this.state.lastActiveGitOp} provider={GitProvider.Azure_devops} gitops="Azure" />
                 </div>
                 <CustomInput autoComplete="off"
                     value={this.state.form.host}
                     onChange={(event) => this.handleChange(event, 'host')}
                     name="Enter host"
                     error={this.state.isError.host}
-                    label="Git Host*"
+                    label={this.state.tab === GitProvider.Azure_devops ? "Azure DevOps Organisation Url*" : "Git Host*"}
                     tabIndex={1}
                     labelClassName="gitops__id form__label--fs-13 fw-5 fs-13" />
                 <div className="mt-16">
                     <CustomInput autoComplete="off" value={this.state.form[key]}
                         tabIndex={2}
                         error={this.state.isError[key]}
-                        label={this.state.tab === GitProvider.Github ? "GitHub Organisation Name*" : "GitLab Group ID*"}
+                        label={this.state.tab === GitProvider.Github ? "GitHub Organisation Name*" : this.state.tab === GitProvider.GitLab ? "GitLab Group ID*" : "Azure DevOps Project Name*"}
```

Comment:
earlier we had gitLabGroupId as default over here we have azure as default. we should have consistent behaviour

Reply:
fixed : GitHub Organisation Name* as default